deploy/terraform/s3.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_s3_bucket" "security_policy_templates" {
```

Comment:
Please add default encryption to the bucket. I know that this content is not super sensitive but it avoids having to have the security team review it.